### ISSUE TITLE
clickablearea grading option Empty

### ISSUE THREAD

**Issue by psmaker** [RunestoneInteractive/RunestoneServer#1026]
When making a .clickablearea problem, it is written and added to an assignment OK, but the grading option is Empty and there is no menu selectable.

![image](https://user-images.githubusercontent.com/28868409/29745244-67a9273a-8a6a-11e7-9bc6-17d4b9630f02.png)

I tried the problem itself and the auto-grade does do something.

It scores a 1 when it is 100% correct. Anything less and it scores a 0. 
Not sure if there is a % correct option on this type of problem.

**Comment by bnmnetp** [comment]
OK, I can see why you don't get any options for clickable areas.  Already fixed on my dev server.  I'll push later today.

**Comment by bnmnetp** [comment]
PS, it appears that for now, the only scoring option that is set up is all or nothing.

**Comment by psmaker** [comment]
I tried going back to my 'New Exercises' assignment to change this scoring option to 'all or nothing'. 

Now, my I hit a new issue - the 'New Exercise' shows up blank - with none of the problems assigned, nor does the Description change to the correct value.

I can't see the clickable problem anymore - nor any other problems previously in the 'New Exercises'.

![image](https://user-images.githubusercontent.com/28868409/29745529-173f596a-8a72-11e7-843c-223bf99cca38.png)

I made a 'New Exercises 2' with all of the old reading problems and when I did, all the grading option and which-to-grade fields were gone.

![image](https://user-images.githubusercontent.com/28868409/29745539-5bfdf782-8a72-11e7-8d72-bd4da7a23e3c.png)
